bkp_composer.lock

```diff
@@ -0,0 +1,5501 @@
+{
```

Comment:
Accidental commit? We used to not commit locks for libraries :v:

Reply:
It was added accidentally.